pages/floor/[index].js

```diff
@@ -0,0 +1,48 @@
+import MoneyDisplay from "~/src/components/MoneyDisplay";
+import Product from "~/src/components/Product";
+import styled from "styled-components";
+import Navigation from "~/src/components/Navigation";
+import { useRouter } from "next/router";
+
+export default function HomePage() {
+  const router = useRouter();
+  const { index } = router.query;
+
+  if (index === "1") {
+    return (
+      <>
+        <MoneyDisplay />
+        <StyledMain>
+          <Product key="tier1" tierId="tier1" />
+          <Product key="tier2" tierId="tier2" />
+          <Product key="tier3" tierId="tier3" />
+          <Product key="tier4" tierId="tier4" />
+          <Product key="tier5" tierId="tier5" />
+        </StyledMain>
+        <Navigation />
```

Comment:
This is just a suggestion✌️
You could think about using a `Layout component` that includes the common elements, such as
- `header` in your case the `MoneyDisplay Component`
- `footer` in your case the `Navigation Component` 

👀 See Next.js [Layout Component](https://nextjs.org/learn/basics/assets-metadata-css/layout-component)

Reply:
Yes ma'am